packages/indexer/src/api/index.ts

```diff
@@ -67,14 +70,36 @@ export async function run(db: DB, logger: Logger) {
     next();
   };
 
+  // Kludge. I don't know how to prevent promBundle from exposing metricsPath
+  const metricsPath = '/' + randomString(20); // '/metrics'
```

Comment:
Is there a way to put the metrics path on a separate port so that it's no exposed?

Reply:
This is something we can improve as we go along